CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.2.2
+January 31, 20189
+### Bug fixes
+* fix(eventtagsutils) : don't exclude falsy values ([#132](https://github.com/optimizely/csharp-sdk/pull/132))
```

Comment:
Lets try and be more descriptive here.

The PR does talk about it, but we can probably give more context here or in the PR.